crates/db_views_actor/src/community_view.rs

```diff
@@ -111,9 +111,14 @@ fn queries<'a>() -> Queries<
 
     if let Some(search_term) = options.search_term {
       let searcher = fuzzy_search(&search_term);
-      query = query
-        .filter(community::name.ilike(searcher.clone()))
-        .or_filter(community::title.ilike(searcher))
+      let name_filter = community::name.ilike(searcher.clone());
+      let title_filter = community::title.ilike(searcher.clone());
+      let description_filter = community::description.ilike(searcher.clone());
+      query = if options.title_only.unwrap_or_default() {
+        query.filter(name_filter.or(title_filter))
```

Comment:
Communities have names, titles, and descriptions. I'm treating `title_only` to mean what it did before, the name or the title.

Reply:
You could directly make a variable `let name_or_title_filter` but this is also fine.